pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -12,7 +12,7 @@ func Platform() (*ovirt.Platform, error) {
 	p := ovirt.Platform{}
 
 	ovirtConfig, err := NewConfig()
-	if err != nil {
+	if err != nil || !ovirtConfig.Validate() {
```

Comment:
It does not appear that `ovirtConfig.Validate` is any different than the test that is performed a few lines down from here.

Reply:
Hey @staebler thank you very much for your help with this issue. I'm going to push a proposed fix, but since I don't have a fully working ovirt setup yet please allow me until next week to test it properly.